src/featureFlags/availableFlags.ts

```diff
@@ -0,0 +1,37 @@
+import { FlagContent } from "./FlagContent";
+
+const AVAILABLE_FLAGS = [
```

Comment:
I think available flags should come from ENV

Reply:
I think it could be even a separated config, env are not enough for store quite sophisticated information (it's more than primitive values)

We are definite flags that can be used in the code (in this array), which also can be stored within the metadata in the future. Using these definition we are able to display some user-friendly information on the UI. Let's start with it for now.

Reply:
Please remember that flags are _not_ configuration, while its often tempting to model them like that. And they should be temporary - so likely flags are very simple strings or booleans. After a feature is merged, they are gone. I hope we don't overengineer them.

What is this non-primitive value you need?

Reply:
I'm renaming this to "payload"